src/index.js

```diff
@@ -1,6 +1,63 @@
+/* eslint-disable*/
 import './styles/main.css';
-import { displayTodoList } from './app.js';
+import {createTask, readTask, updateTask, deleteTask,} from './app.js';
 
-document.addEventListener('DOMContentLoaded', () => {
-  displayTodoList();
+const todoInput = document.querySelector('#todo-input');
+const todoList = document.querySelector('#todo-list');
+const deleteBtn = document.querySelector('.delet-btn');
+
+function renderTasks() {
+  const tasks = readTask();
+  todoList.innerHTML = '';
+  tasks.forEach((task, index) => {
+    const li = document.createElement('li');
+    li.innerHTML = `
+      <label>
+        <input type="checkbox" class="task-checkbox" data-index="${index}">
+        <span class="task-text ${task.completed ? 'completed' : ''}">${task.text}</span>
+      </label>
+      <button type="button" class="update-btn" id="update" data-index="${index}"><i class='fas fa-edit'></i></button>
```

Comment:
- [ ] You have done a great job 💪making the project work. However, using a single JavaScript file for all functionalities is not correct 🙆‍♀️ as per the project requirements. You should create a new JavaScript file for each new functionality. 🙏Please check the screenshot below for the project requirements.
![image](https://user-images.githubusercontent.com/63445083/236112527-9acbb32b-bfe7-4dc2-a0b4-5f350c29c0a2.png)
To implement this, you can create a new folder📂 called modules and add all new JavaScript files according to their functionalities, such as AddTodo.js, EditTodo.js, and DeleteTodo.js. After that, you can import these files into your index.js file. This will improve your coding style by using modular code, which is considered the best✔ practice.